plugins/modules/swim_workflow_manager.py

```diff
@@ -620,6 +703,46 @@ def get_image_id(self, name):
 
         return image_id
 
+    def get_cco_image_id(self, cco_image_name):
+        """
+    Retrieve the unique image ID from Cisco.com based on the provided image name.
```

Comment:
Indentation is not correct for this block.

Reply:
updated the code